incubator/zookeeper/values.yaml

```diff
@@ -2,16 +2,23 @@
 # This is a YAML-formatted file.
 # Declare name/value pairs to be passed into your templates.
 # name: value
-
-
-Name: zk
-PeerPort: 2888
+Servers: 3
+Cpu: "1"
+Memory: "4Gi"
+Heap: "2G"
+Storage: "50Gi"
+StorageClass : "default"
```

Comment:
Needs to be uncommented. Otherwise PVCs arent being bound by default (at least in GKE).